### DISCUSSION TITLE
In the revealjs environment, is it possible to change the font size with the keyboard

### DISCUSSION THREAD

**Discussion by ChoCho66** [quarto-dev/quarto-cli#6179] [Q&A] [answered]
### Description

As in the title, is it possible to add something like in <https://www.w3.org/Talks/Tools/Slidy2/#(2)>

> - Font sizes automatically adapt to browser window size
>   - use S and B keys for manual control (or < and >, or the - and + keys on the number pad

this function in the revealjs environment?

So that I can immediately modify the font size when there is a problem with the text layout.

**Comment by mcanouil** [comment, marked as answer]
That's not a question for Quarto but for Reveal.js.
See <https://revealjs.com/> and <https://github.com/hakimel/reveal.js>.

You could still implement some JavaScript but it might get tricky since Reveal.js is JavaScript already.
